extensions/wikia/Recirculation/RecirculationController.class.php

```diff
@@ -20,6 +21,7 @@ public function index() {
 		$posts = $fandomDataService->getPosts( $this->type );
 
 		if ( count( $posts ) > 0 ) {
+			$this->response->setCacheValidity( self::CACHE_DURATION );
```

Comment:
and I think `setCacheValidity` is actually dealing with Varnish/Browser cache, right?